package.json

```diff
@@ -16,18 +17,38 @@
   ],
   "dependencies": {},
   "devDependencies": {
+    "ava": "^0.25.0",
     "eslint": "^4.7.1",
+    "husky": "^0.14.3",
+    "lint-staged": "^6.0.1",
+    "lodash.clonedeep": "^4.5.0",
+    "prettier": "^1.10.2",
     "rollup": "^0.50.0",
     "uglify-js": "^3.1.1"
   },
-  "description": "Polyfill for smooth scroll behavior",
   "main": "dist/smoothscroll.js",
   "scripts": {
-    "test": "eslint src/**/*.js",
+    "lint": "eslint {src,test}/**/*.js",
+    "pretest": "yarn lint",
```

Comment:
I really don’t want this in `pretest`. If we want to run lints on test do it in `posttest`. No reason to make someone fix something nitpicky like spacing or semi-colons to see if tests pass.